Duplicate variable declarations in `extensions/vscode/src/activation/activate.ts`

### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: macOS 14.5
- Continue: 0.9.140
- IDE: VSCode
```


### Description

[extensions/vscode/src/activation/activate.ts](https://github.com/continuedev/continue/blob/main/extensions/vscode/src/activation/activate.ts#L12-L20) declares `resolveVsCodeExtension` and `vscodeExtensionPromise` twice. 

This was introduced in [#1356](https://github.com/continuedev/continue/pull/1356/files#diff-ccb2651e83bb926d28195576f7f96320d33a1fa6364c4a0700c2e9db938034bfR11-R15).

### To reproduce

Run the `install-all-dependencies` task.

### Log output

```Shell
Error: Command failed: npx vsce package --out ./build patch --no-dependencies
✘ [ERROR] The symbol "resolveVsCodeExtension" has already been declared

    src/activation/activate.ts:17:4:
      17 │ let resolveVsCodeExtension = (_: VsCodeExtension): void => {};
         ╵     ~~~~~~~~~~~~~~~~~~~~~~

  The symbol "resolveVsCodeExtension" was originally declared here:

    src/activation/activate.ts:12:4:
      12 │ let resolveVsCodeExtension = (_: VsCodeExtension): void => {};

✘ [ERROR] The symbol "vscodeExtensionPromise" has already been declared

    src/activation/activate.ts:18:13:
      18 │ export const vscodeExtensionPromise: Promise<VsCodeExtension> = ne...
         ╵              ~~~~~~~~~~~~~~~~~~~~~~

  The symbol "vscodeExtensionPromise" was originally declared here:

    src/activation/activate.ts:13:13:
      13 │ export const vscodeExtensionPromise: Promise<VsCodeExtension> = ne...
```

--- a/binary/build.js
+++ b/binary/build.js
@@ -51,7 +51,7 @@ async function installNodeModuleInTempDirAndCopyToCurrent(packageName, toCopy) {
   console.log(`Copying ${packageName} to ${toCopy}`);
   // This is a way to install only one package without npm trying to install all the dependencies
   // Create a temporary directory for installing the package
-  const adjustedName = packageName.replace(/^@/, "").replace("/", "-");
+  const adjustedName = toCopy.replace(/^@/, "").replace("/", "-");
   const tempDir = path.join(
     __dirname,
     "tmp",

--- a/extensions/vscode/scripts/prepackage.js
+++ b/extensions/vscode/scripts/prepackage.js
@@ -316,7 +316,7 @@ const exe = os === "win32" ? ".exe" : "";
     console.log(`Copying ${packageName} to ${toCopy}`);
     // This is a way to install only one package without npm trying to install all the dependencies
     // Create a temporary directory for installing the package
-    const adjustedName = packageName.replace(/^@/, "").replace("/", "-");
+    const adjustedName = toCopy.replace(/^@/, "").replace("/", "-");
 
     const tempDir = `/tmp/continue-node_modules-${adjustedName}`;
     const currentDir = process.cwd();

--- a/extensions/vscode/src/activation/activate.ts
+++ b/extensions/vscode/src/activation/activate.ts
@@ -14,11 +14,6 @@ export const vscodeExtensionPromise: Promise<VsCodeExtension> = new Promise(
   (resolve) => (resolveVsCodeExtension = resolve),
 );
 
-let resolveVsCodeExtension = (_: VsCodeExtension): void => {};
-export const vscodeExtensionPromise: Promise<VsCodeExtension> = new Promise(
-  (resolve) => (resolveVsCodeExtension = resolve),
-);
-
 export async function activateExtension(context: vscode.ExtensionContext) {
   // Add necessary files
   getTsConfigPath();

